doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -156,48 +156,15 @@ In order for the networking solution(s) to be conformant with the Reference Arch
 
 |Ref|Specification|Details|Requirement Trace|
 |---|---|---|---|
-|`ra2.ntw.001`||||
-|`ra2.ntw.002`||||
-|`ra2.ntw.003`||||
-|`ra2.ntw.004`||||
+|`ra2.ntw.001`|CNI multiplexer/metaplugin|As the selected CNI multiplexer/metapulgin MUST support other CNI plugins (`req.inf.ntw.06`) and SHOULD provide an API based solution to administer the networks from a central point (`req.inf.ntw.03`) the selected CNI multiplexer/metapulgin must be [DANM](https://github.com/nokia/danm).| [req.inf.ntw.06](./chapter02.md#23-kubernetes-architecture-requirements), [req.inf.ntw.03](./chapter02.md#23-kubernetes-architecture-requirements) |
```

Comment:
Well, we had a comparison between Multus and DANM table in the previous format. I can add it also to here. I did not add Knitter there as it is not active and I have no view about its features. What CNI integration you mean on OVN?  

In my view it matters if the deployment artefacts are different as this means that all the variants of these artefacts should be tested and this is exactly what we would like to avoid.

Reply:
I've added the comparison table back.

Reply:
Again, in a RA specification, the mechanisms by which multiple interfaces can be achieved should be described and what capabilities MUST be present. A specific RI could very well not leverage a metaplugin and still be able to support multi-interfaces through CRDs or NSM constructs.

Reply:
> Again, in a RA specification, the mechanisms by which multiple interfaces can be achieved should be described and what capabilities MUST be present, but NOT dictating one. A specific RI could very well not leverage a metaplugin and still be able to support multi-interfaces through CRDs or NSM constructs.

This is why I think we need to be clear on the tradeoffs and properly document them (I know we've talked about them ad nauseam but we've not written them down).  e.g. 
- If we specify a particular metaplugin is required then we close of all other solutions to the requirement, but we have a tight and consistent set of deployment patterns). There is potentially lower cost to the software vendor as the deployment artefacts are consistent.  The downsides are potentially higher cost for platform vendors to bring e.g. DANM into their platform, and we are unable to use these other solutions as described.
- If we don't specify a particular metaplugin is required, then we allow for multiple methods of achieving the requirement, but we are likely to end up with a software vendor needing to create multiple deployment methods for their software - i.e. a helm chart for VNFa using Multus, a helm chart for VNFb using DANM, a helm chart for VNFc using something else.  So the platform vendors or operators if they're rolling their own can get away with choosing one of those things to bundle with their platform, so potentially lower cost to them.  But the downside is that the software vendors end up needing to provide artefacts for multiple networking solutions.

@CsatariGergely is correct that we have verbally agreed that the first approach is should be taken.  I'm not convinced that we have all made this agreement with full view of the pros/cons.

Reply:
However I think the missing of CNTT is _not_ to enable options where they affect the workload or its artifacts, we can collect and discuss the tradeoffs.

but

Let's do it in a different pr not in this content transformation one. @tomkivlin , @dabernie please create an issue if you would like to open the discussion.

Reply:
I have added some requirements in RA2 Ch02 (PR #1872 ) that also need to be taken into account once we have all agreed to their need

Reply:
> Let's do it in a different pr not in this content transformation one. @tomkivlin , @dabernie please create an issue if you would like to open the discussion.

Done - #1874

Reply:
Sorry could not create issue was on PTO